ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -737,21 +737,26 @@
      (checkbox-container update! table? label warning error style checked?)]
     (checkbox-container update! table? label warning error style checked?)))
 
-(defmethod field :checkbox-group [{:keys [update! table? label show-option options help]} data]
+(defmethod field :checkbox-group [{:keys [update! table? label show-option options help error warning header?]} data]
   (let [selected (set (or data #{}))]
     [:div.checkbox-group
-     [:h4 (stylefy/use-style style-form-fields/checkbox-group-label) label]
+     (when header?
+       [:h4 (stylefy/use-style style-form-fields/checkbox-group-label) label])
```

Comment:
This label has been made optional under, but existing usages of `:checkbox-group` fields have not been updated. Is that intentional?